### PR TITLE
Antelope CAIP Entry

### PR DESCRIPTION
**Pull request by porkchop**
Antelope (formerly EOSIO) CAIP entry. Should also deprecate https://github.com/ChainAgnostic/namespaces/tree/feat/add-eosio

### CODE REVIEW

**Comment by porkchop** [comment]
Am also looking for feedback from the community...

File: antelope/README.md
Comment (bumblefudge) [inline antelope/README.md:8]:
```suggestion
```

**Comment by bumblefudge** [comment]
@nsjames nothing in here looks controversial but maybe if you help @porkchop with a careful review and +1 they'll return the favor next month when we've got a CAIP-10 and CAIP-19 profile as well? :D

**Comment by porkchop** [comment]
> @nsjames nothing in here looks controversial but maybe if you help @porkchop with a careful review and PR approval they'll return the favor next month when we've got a CAIP-10 and CAIP-19 profile as well? :D

We certainly will :) Happy to include that as well in the next month

File: antelope/README.md
Comment (porkchop) [reply antelope/README.md]: Fixed!

**Comment by porkchop** [comment]
FYI @bumblefudge , at the request of the community, I added test cases for the major chains in the ecosystem as well. Also thank you kindly for your help and attention!

**Comment by bumblefudge** [comment]
Seems fine, waiting on a review/+1 from other players in the space.  My only nagging confusion is:
screengrab of references section
It's a little confusing why the namespace is named after the codebase (Antelope) when `eos` or `eosio` would have more name recognition; perhaps others would not even think to look in the `antelope` namespace if they are asking to research "eos"?  I'll leave it up to you and your reviewers from the ecosystem to debate, but my only suggestion would be to use the body text in readme.md file and/or the TITLE frontmatter property of same to make sure people who've only heard of EOS realize that EOS is covered in the `antelope` space.

Note, the TITLE frontmatter is what appears on the "table of contents" at namespaces.chainagnostic.org , so this is maybe the most important place to catch people's eye who may not be familiar with the space, like so:
screengrab of table of contents

File: antelope/README.md
Comment (bumblefudge) [inline antelope/README.md:3]:
```suggestion
title: Antelope and EOS Namespace
```
or
```suggestion
title: Antelope (formerly known as EOS) Namespace
```
or something else that telescopes to unfamiliar readers the scope

File: antelope/README.md
Comment (bumblefudge) [inline antelope/README.md:13]:
```suggestion
the Antelope ecosystem, which include EOS, Telos, Wax, and more.
```

**Review by bumblefudge — APPROVED** [review]

**Comment by porkchop** [comment]
> Seems fine, waiting on a review/+1 from other players in the space. My only nagging confusion is: screengrab of references section It's a little confusing why the namespace is named after the codebase (Antelope) when `eos` or `eosio` would have more name recognition; perhaps others would not even think to look in the `antelope` namespace if they are asking to research "eos"? I'll leave it up to you and your reviewers from the ecosystem to debate, but my only suggestion would be to use the body text in readme.md file and/or the TITLE frontmatter property of same to make sure people who've only heard of EOS realize that EOS is covered in the `antelope` space.
> 
> Note, the TITLE frontmatter is what appears on the "table of contents" at namespaces.chainagnostic.org , so this is maybe the most important place to catch people's eye who may not be familiar with the space, like so: screengrab of table of contents

Yes great suggestions, thank you. I'll double confirm with the community and get back.

File: antelope/README.md
Comment (porkchop) [reply antelope/README.md]: I followed your second suggestion

File: antelope/README.md
Comment (porkchop) [reply antelope/README.md]: Thank you. Change made

**Comment by porkchop** [comment]
I've made the suggested changes with agreement  from the community. I'll get one or more to confirm support

**Comment by nsjames** [comment]
This is looking great, and I'm loving the discussion around it in the various antelope communities 🔥

I'm trying to stay away a bit because I represent EOS but I think this turned into a much larger initiative than was maybe originally intended, which is fantastic. The edits to showcase some of the history are excellent

**Comment by aaroncox** [comment]
Commenting to confirm that everything looks accurate to me!

**Comment by porkchop** [comment]
Thanks @bumblefudge and everyone for the help

**Pull request merged**
